velox/docs/monthly-updates/august-2022.rst

```diff
@@ -0,0 +1,80 @@
+******************
+August 2022 Update
+******************
+
+Documentation
+=============
+
+* Add documentation for :func:`combinations` Presto function.
+* Add documentation for :doc:`/functions/window`.
+* Add documentation for :func:`array_sort`.
+* Add `website`_ for Velox.
+
+.. _website: https://velox-lib.io
+
+Core Library
+============
+
+* Add support for Right Semi Join.
+* Add support for a simple in-memory Window operator.
+* Add support for `month` string in Joda library.
```

Comment:
Is there a way to describe it in a way that is more user-friendly, e.g. what does it mean for a user? I assume this enables some new format specifying in the parse_datetime function. If so, we can say that and move it to Presto functions section.